includes/manifest.inc

```diff
@@ -11,94 +11,145 @@
 function islandora_iiif_manifests_view_manifest_json(AbstractObject $object) {
   module_load_include('inc', 'islandora_iiif_manifests', 'includes/utilities');
   module_load_include('inc', 'islandora', 'includes/datastream');
-
+  
+  $data = "";
+  
   if (isset($object[ISLANDORA_IIIF_MANIFESTS_DATASTREAM_ID]) && islandora_datastream_access(ISLANDORA_VIEW_OBJECTS, $object[ISLANDORA_IIIF_MANIFESTS_DATASTREAM_ID])) {
     header('Access-Control-Allow-Origin: *');
-
+    
     islandora_view_datastream($object[ISLANDORA_IIIF_MANIFESTS_DATASTREAM_ID]);
   }
   else {
-    islandora_iiif_manifests_create_json_response('403 Forbidden', 'Forbidden');
+    if (!isset($object[ISLANDORA_IIIF_MANIFESTS_DATASTREAM_ID]))
```

Comment:
Always use { } with if statements, even if it only has 1 line. It is more readable that way. So should be:
if (!isset($object[ISLANDORA_IIIF_MANIFESTS_DATASTREAM_ID])) {
  $data = "No manifest found";
}
if (islandora_datastream_access(ISLANDORA_VIEW_OBJECTS, $object[ISLANDORA_IIIF_MANIFESTS_DATASTREAM_ID])) {
  $data = "Forbidden";
}